Resources/Prototypes/DeltaV/Entities/Objects/Weapons/Guns/Rifles/rifles.yml

```
@@ -0,0 +1,141 @@
+- type: entity
+  name: vulcan
+  parent: [BaseWeaponRifle, BaseGunWieldable]
+  id: WeaponRifleVulcan
+  description: One of the heaviest small arms to grace Security's armory, this rifle is a modern take on a classic, informally dubbed the "Right Arm of the Free World". Uses .30 rifle ammo.
+  components:
+  - type: Sprite
+    sprite: DeltaV/Objects/Weapons/Guns/Rifles/vulcan.rsi
+    layers:
+      - state: base
+        map: ["enum.GunVisualLayers.Base"]
+      - state: mag-0
+        map: ["enum.GunVisualLayers.Mag"]
+  - type: GunWieldBonus
+    minAngle: -20
+    maxAngle: -30
+  - type: Gun
+    minAngle: 21
+    maxAngle: 54
+    angleIncrease: 6
+    angleDecay: 24
+    fireRate: 5
+    selectedMode: SemiAuto
+    availableModes:
+    - SemiAuto
+    - FullAuto
+  - type: ChamberMagazineAmmoProvider
+  - type: ItemSlots
+    slots:
+      gun_magazine:
+        name: Magazine
+        startingItem: MagazineLightRifle
+        insertSound: /Audio/Weapons/Guns/MagIn/ltrifle_magin.ogg
+        ejectSound: /Audio/Weapons/Guns/MagOut/ltrifle_magout.ogg
+        priority: 2
+        whitelist:
+          tags:
+            - MagazineLightRifle
+      gun_chamber:
+        name: Chamber
+        startingItem: CartridgeLightRifle
+        priority: 1
+        whitelist:
+          tags:
+            - CartridgeLightRifle
+  - type: ContainerContainer
+    containers:
+      gun_magazine: !type:ContainerSlot
+      gun_chamber: !type:ContainerSlot
+  - type: MagazineVisuals
+    magState: mag
+    steps: 1
+    zeroVisible: true
+  - type: Appearance
+  - type: Clothing
+    sprite: DeltaV/Objects/Weapons/Guns/Rifles/vulcan.rsi
+  - type: StaticPrice
+    price: 1300
+
+- type: entity
+  name: ceremonial rifle
+  parent: [BaseWeaponSniper, BaseC2ExpeditionContraband]
+  id: WeaponSniperCeremonial
+  description: A ceremonial variant of the Mark 1 Rifle, in tasteful blue and white. Uses .30 rifle ammo.
+  components:
+  - type: Sprite
+    sprite: DeltaV/Objects/Weapons/Guns/Rifles/ceremonial_grand.rsi
+  - type: Clothing
+    sprite: DeltaV/Objects/Weapons/Guns/Rifles/ceremonial_grand.rsi
+  - type: Gun
+    fireRate: 2
+    selectedMode: SemiAuto
+    availableModes:
+    - SemiAuto
+    soundGunshot:
+      path: /Audio/Weapons/Guns/Gunshots/rifle2.ogg
+  - type: BallisticAmmoProvider
+    capacity: 5
+    proto: CartridgeLightRifle
+    whitelist:
+      tags:
+      - CartridgeLightRifle
+
+
+- type: entity
+  name: CAWS-25 Jackdaw
+  parent: [BaseWeaponRifle, BaseGunWieldable, BaseC3Contraband]
+  id: WeaponRifleJackdaw
+  description: The beginning of the end is heralded by the song of a Jackdaw. Uses .25 caseless ammo.
+  components:
+  - type: Item
+    size: Large
+  - type: Sprite
+    sprite: DeltaV/Objects/Weapons/Guns/Rifles/jackdaw.rsi
+    layers:
+      - state: base
+        map: ["enum.GunVisualLayers.Base"]
+      - state: mag-0
+        map: ["enum.GunVisualLayers.Mag"]
+  - type: Gun
+    minAngle: 2
+    maxAngle: 6
```

These numbers are busted.  Remember that BaseGunWieldable defined a GunWieldBonus of -20/-30 by default.

If these should be the ending values, consider the following suggestion:

```suggestion
    minAngle: 22
    maxAngle: 36
```